.github/workflows/e2e.yaml

```
@@ -65,14 +65,19 @@ jobs:
         openshift_password: ${{ secrets.OPENSHIFT_PASSWORD }}
         insecure_skip_tls_verify: true
         namespace: default
-    - run: make deploy
+    - run: |
```

We should probably give a `name:` to this step.
```suggestion
    - name: Force delete all SpecialResources
      run: |
```

---

Done